cdk/script/export-monitoring-file

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+ROOT_DIR="${DIR}/../.."
+INDEX_JS_BUILT_FILE="${ROOT_DIR}/monitoring/dist/index.js"
+
+get_absolute_path_directory() {
+  # $1 : relative filename
+  absolute_src_file_path="$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"  # Gets the absolute path of the index.js file
+  echo $(dirname "$absolute_src_file_path") # Get directory name
+}
+
+if test -f "${INDEX_JS_BUILT_FILE}"; # Check if index js file is in the monitoring file
+then
+	src_file=$(find $INDEX_JS_BUILT_FILE) # Gets the index.js file generated in the monitoring/dist. Throws error if it doesn't exist
+	absolute_src_directory=$(get_absolute_path_directory "$src_file") # Gets the directory of the index.js file monitoring/dist
+
+	echo "Zipping Built Files"
+	cd "${absolute_src_directory}" # Go to monitoring/dist folder
+	zip -FSr "${ROOT_DIR}/cdk/dist/cmp-monitoring-lambda.zip" "." # Zip the monitoring/dist folder into cdk/dist
+	echo "File was successfully zipped ${region}"
```

Comment:
Is region still required here if we're no longer needing that?

Reply:
Thanks for the spot.